src/ng/directive/input.js

```diff
@@ -2142,7 +2142,7 @@ var ngValueDirective = function() {
    *  makes it possible to use ngValue as a sort of one-way bind.
    */
   function updateElementValue(element, attr, value) {
-    element.prop('value', value);
+    element.prop('value', angular.isUndefined(value) ? null : value);
```

Comment:
Leaving out `angular.` resulted in a broken build. But it looks like it's not related to that ?